sections/leaderboard/Leaderboard/Leaderboard.tsx

```diff
@@ -30,14 +29,13 @@ type Stat = {
 	totalVolume: Wei;
 };
 
-const Leaderboard: FC<LeaderboardProps> = ({ compact }: LeaderboardProps) => {
+const Leaderboard: FC<LeaderboardProps> = ({ compact, searchTerm }: LeaderboardProps) => {
 	const { t } = useTranslation();
 
-	const [searchTerm, setSearchTerm] = useState<string | null>();
-
 	const walletAddress = useRecoilValue(walletAddressState);
 	
 	const statsQuery = useGetStats();
+	console.log(statsQuery)
```

Comment:
Looks like you might have forgotten this here.

Reply:
good catch, removed.